### ISSUE TITLE
[BUG] SfFooterColumn invalid html issues

### ISSUE THREAD

**Issue by DavidGolodetsky** [vuestorefront/storefront-ui#2102]
**Describe the bug**
Nuxt html-validator throws two errors on the SfFooterColumn component:
1. Element `<div>` is not permitted as content in `<button>`
2. `<button>` is missing required "type" attribute

**How to reproduce**
You can try to put the output of the component to the [W3C validator](https://validator.w3.org/#validate_by_input+with_options)

**Expected behavior**
The SfFooterColumn component would be restructured in a way that HTML would be valid

**Actual behavior**
Each column title is a button tag without the type "button" and it has a div as a child.

**Screenshots**
![image](https://user-images.githubusercontent.com/32278933/141163759-4eedc6f0-dead-4b91-8343-082fab893c42.png)

**Additional context**
Related to #2097

**Comment by justyna-13** [comment]
Hi @DavidGolodetsky, regarding the first error - div inside button - this was fixed in #2062 so I hope it's no longer in our newest version :) when it comes to the second error - required button "type" attribute - according to HTML spec type attribute isn't required (as a default value is "submit"). Of course, it's possible to change SfButton type attribute by binding another type

**Comment by DavidGolodetsky** [comment]
Great, thank you! 
Regarding the button type issue, I've touched on this topic [here](https://github.com/vuestorefront/storefront-ui/issues/2103#issuecomment-980090094). 
I see a potential issue with SfButton submit type as default:
You can pass to this component when you're using it as an Atom component. But what if you're using it as a part of a molecule or organism component and you need to wrap this component in a form tag. Then you'd have to prevent submit event from the SfButton component.
